### ISSUE TITLE
Image::make dies silently with images created from Android 5.1.1 device

### ISSUE THREAD

**Issue by sarobinson9535** [Intervention/image#450]
Hello all, 

I have been using the Intervention package successfully now for quite some time, and its been rock solid so far.  This week my Android device upgraded to 5.1.1, after this, my mobile app quit being able to upload photos that are passed through to the back end and processed via Intervention Image commands.  

Specifically the Image::make command doesn't complete, or throw an error, when the file (attached) is delivered.  I modified the code to write out the file initially to ensure the file got to the server correctly, and it does.

The file is available, a browser can render it successfully, and if you store it on the device, a mobile device can also render it successfully.  Even the Android Studio debugger can render the image.  I  dumped all the file attributes I have visibility to see, and the file looks identical to previous Android versions that work, so I'm a bit stumped at this point and hoping for some direction.

Thanks
Scott

![rearfacingimage](https://cloud.githubusercontent.com/assets/11712850/10854756/b3c3873a-7f0b-11e5-815d-6fbe779599aa.jpg)

**Comment by olivervogel** [comment]
That's strange. I just tried to load your image and got no problems.

```php
$img = Image::make('b3c3873a-7f0b-11e5-815d-6fbe779599aa.jpg');
```

If PHP dies silently, it often runs out of memory. Did you check your [memory settings](http://php.net/manual/en/ini.core.php#ini.memory-limit)?

**Comment by stueynet** [comment]
+1 on this one. Android device. Consistently fails. If I re-export the image again from desktop with same dimensions and filesize, works just fine. Head exploding on this.

**Comment by stueynet** [comment]
Did some additional tests. First off raised up the memory limit and also started using $file->getRealPath() and still having the same issue. 
```
Intervention\Image\Exception\NotReadableException·Unable to read image from file (/tmp/phpMAMExa).
Full TraceRaw
app/Http/Controllers/Api/V1/UploadsController.php:300App\Http\Controllers\Api\V1\UploadsController::storeImage

$image = Image::make($file->getRealPath());
```
It happens specifically when users take a photo with an Android device and try and upload it. All other files work properly no mater what size or dimensions. Also if I take the exact image and simply resave it on my computer, it uploads just fine.

**Comment by andrew-za** [comment]
Same issue here, I get the following error on mobile:
NotReadableException in Decoder.php line 46:
Unable to read image from file (...tmp\php6586.tmp)

**Comment by bachras** [comment]
I am also facing this issue. Does anybody know how to solve? Thanks

**Comment by stueynet** [comment]
Yeah @bachras I put this in the constructor of my uploads controller:

```php
    public function __construct()
    {
        /* Required to deal with:
         * https://github.com/Intervention/image/issues/21
         * https://github.com/owncloud/core/pull/21950
         * Example corrupt image:
         * https://storage.googleapis.com/go-attachment/3916/2/xx.jpg
        */

        ini_set("gd.jpeg_ignore_warning", 1);
    }
```
